packages/frontend/src/components/composer/menuAttachment.tsx

```diff
@@ -36,28 +39,59 @@ export default function MenuAttachment({
   const openConfirmationDialog = useConfirmationDialog()
   const { sendVideoChatInvitation } = useVideoChat()
   const { openDialog, closeDialog } = useDialog()
+  const { sendMessage } = useMessage()
   const [settings] = useStore(SettingsStoreInstance)
   const accountId = selectedAccountId()
 
+  const confirmSendMultipleFiles = (
+    filePaths: string[],
+    msgViewType: Viewtype
+  ) => {
+    if (!selectedChat) {
+      throw new Error('no chat selected')
+    }
+    openDialog(ConfirmSendingFiles, {
+      sanitizedFileList: filePaths.map(path => ({ name: basename(path) })),
+      chatName: selectedChat.name,
+      onClick: async (isConfirmed: boolean) => {
+        if (!isConfirmed) {
+          return
+        }
+
+        for (const filePath of filePaths) {
+          sendMessage(accountId, selectedChat.id, {
+            file: filePath,
+            viewtype: msgViewType,
+          }).then(() => {
+            // start sending other files, don't wait until last file is sent
+            runtime.removeTempFile(filePath)
```

Comment:
The line 
`runtime.removeTempFile(filePath)`
is definitely wrong since as Simon says, filePath is the original file path

"Why do we have to do that there," ?
The reason is that drag & drop only provides [DataTransfer files](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/files) which have no path and we need to pass a local path as file attachement to core

Reply:
it is needed for browser, for electron the call just failed because it can not delete non-temp files.